### PR TITLE
docs(domains&dns): how to transfer external domain MTA-2503

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: network/domains-and-dns/how-to/transfer-external-domain.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Click **Domains and DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Domains and DNS dashboard displays. Click **Register a new domain**:
+    <Lightbox src="scaleway-register_new_domain.webp" alt="" />
+2. Enter the domain name you wish to transfer in the box and click the search icon to verify the domain is available. A selection of available domain names displays.
+3. Click **Transfer** next to the domain name you wish to transfer. A screen displays to help you prepare your transfer.
+    <Message type="important">
+        - You must have administrative rights to manage and unlock your domain for transfers.
+        - You must have the authorization code to initiate the transfer.
+        - Your domain must have been registered at least 60 days ago or renewed for more than 45 days.
+        - Your domain expires in at least 2 weeks.
```
Comment (RoRoJ) [inline network/domains-and-dns/how-to/transfer-external-domain.mdx:53]:
```suggestion
        - Your domain must not expire for a minimum of 2 weeks.
```

File: network/domains-and-dns/how-to/transfer-external-domain.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    </Message>
+4. Click **Initiate transfer** once you are ready to proceed.
+5. Enter your authorization code and click **Configure registrant contact**.
+    <Message type="tip">
+     You can find your authorization code under your domain information or by asking for it directly to your current registrar.
```
Comment (RoRoJ) [inline network/domains-and-dns/how-to/transfer-external-domain.mdx:58]:
```suggestion
     You can find your authorization code under your domain information or by asking your current registrar for it directly.
```

**Review by RoRoJ — APPROVED** [review]

File: network/domains-and-dns/how-to/transfer-external-domain.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have entered your credit card information on your account
+  - You have external domains
+  - You have requested the authorization code to transfer your domain from your registrar
+</Message>
+
+## How to transfer .fr and .be domains
+
+1. Click **Domains and DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Domains and DNS dashboard displays. Click **Register a new domain**:
```
Comment (mid1221213) [inline network/domains-and-dns/how-to/transfer-external-domain.mdx:27]: Same comment as above → "Register new domain" without the "a".

File: network/domains-and-dns/how-to/transfer-external-domain.mdx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+5. Enter your authorization code and click **Validate**. Your order summary displays.
+    <Message type="tip">
+     You can find your authorization code under your domain information or by asking for it directly to your current registrar.
+    </Message>
+6. Check the box confirming you accept to be debited.
+7. Click **Transfer domain**. Once your domain transfer is approved, your domain will display in your **Internal Domains** tab.
+
+## How to transfer domains with other extensions
+
+1. Click **Domains and DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Domains and DNS dashboard displays. Click **Register a new domain**:
```
Comment (mid1221213) [inline network/domains-and-dns/how-to/transfer-external-domain.mdx:47]: Same comment as above → "Register new domain" without the "a".

**Review by mid1221213 — APPROVED** [review]
LGTM

File: network/domains-and-dns/how-to/add-external-domain.mdx [hunk 5]
```diff
@@ -21,7 +21,7 @@ Domains and DNS allows you to manage and register your [internal](/network/domai
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
 </Message>
 
-1. Click **DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Domains and DNS dashboard displays. Click the **External Domains** tab, then click **Manage a new domain**.
+1. Click **Domains and DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Domains and DNS dashboard displays. Click the **External Domains** tab, then click **Manage a new domain**.
```
[the remark this answers is not in the document]
Comment (nedaehm) [reply network/domains-and-dns/how-to/add-external-domain.mdx:24]: I fixed it in another PR 👍🏼

**Pull request merged**